src/routes/github/create-branch/github-create-branch-post.ts

```diff
@@ -92,7 +92,7 @@ export const GithubCreateBranchPost = async (req: Request, res: Response): Promi
 		};
 		statsd.increment(metricCreateBranch.created, tags);
 	} catch (err) {
-		req.log.error({ err }, "Error creating branch");
+		req.log.error({ err, jiraHost }, "Error creating branch");
```

Comment:
Looks like it is already there...
<img width="300" alt="image" src="https://user-images.githubusercontent.com/20631664/224217479-91c17207-655a-4747-9703-7c67f8b9b02b.png">